apicast/src/oauth/apicast_oauth.lua

```diff
@@ -78,13 +76,13 @@ function _M.respond_and_exit(status, body, headers)
   ngx.exit(ngx.HTTP_OK)
 end
 
-function _M.respond_with_error(status, message)
+function _M.respond_with_error(status, message, description)
```

Comment:
see rfc: https://tools.ietf.org/html/rfc6749#section-5.2 message should really be called error as per the rfc and then you can also have error_description but this latter one is optional